human-tracking-and-pose-detection/main.py

```diff
@@ -1,234 +1,144 @@
-downloadModel=True
-runTestImg=False
+#handle command line arguments
+import argparse
+parser = argparse.ArgumentParser(description='Example command-line parser')
+parser.add_argument('-v', '--verbose', action='store_true', help='Enable print_verbose_only outputs')
+parser.add_argument('-k', '--display_keypoint', action='store_true', help='Enable display of keypoints')
+parser.add_argument('-i', '--display_image', action='store_true', help='Enable display of image and keypoints')
+parser_args = parser.parse_args()
```

Comment:
Parse commandline arguments in a function called from main()

Reply:
solved